config.js

```diff
@@ -247,6 +250,26 @@ function setRankedUnrankedOptionsDefaults(rankedUnrankedOptions, argv) {
     }
 }
 
+function exportValidLogfileFilename(argvLogfile) {
```

Comment:
`exportLogfileFilename` would be detailed enoughn i think

Reply:
sure :)